reconcile/utils/mr/__init__.py

```diff
@@ -46,7 +53,7 @@ def init_from_sqs_message(message):
     # and fail early if that type is not on the map.
     msg_type = message.pop('pr_type')
     if msg_type not in types_map:
-        raise TypeError(f'type {msg_type} no supported')
+        raise UnknownMergeRequestType(f'type {msg_type} no supported')
```

Comment:
TBH, I don't mind a `TypeError` here and keeping fewer classes on our code.

Reply:
i was really thinking about introducing this one. but in the end this to me because a TypeError can come from everywhere while this dedicated error class is a specific hint what happened. opinionated for sure :)